app/views/_alerts.html

```diff
@@ -1,28 +1,16 @@
-<div ng-attr-row="{{toast}}"  ng-attr-wrap="{{toast}}">
-  <div ng-repeat="(alertID, alert) in (alerts | filterCollection : filter) track by (alertID + (alert.message || alert.details))" ng-if="!alert.hidden" class="alert-wrapper animate-repeat" ng-class="{'animate-slide': animateSlide}">
-    <div class="alert word-break" ng-class="{
-      'alert-danger': alert.type === 'error',
-      'alert-warning': alert.type === 'warning',
-      'alert-success': alert.type === 'success',
-      'alert-info': !alert.type || alert.type === 'info',
-      'toast-pf': toast,
-      'mar-left-sm': toast
-    }">
-      <button ng-if="!hideCloseButton" ng-click="close(alert)" type="button" class="close">
+<div ng-attr-row="{{$ctrl.toast}}"  ng-attr-wrap="{{$ctrl.toast}}">
+  <div ng-repeat="(alertID, alert) in ($ctrl.alerts | filterCollection : filter) track by (alertID + (alert.message || alert.details))" ng-if="!alert.hidden" class="alert-wrapper animate-repeat" ng-class="{'animate-slide': $ctrl.animateSlide}">
+    <div class="alert word-break" ng-class="{ 'alert-danger': alert.type === 'error', 'alert-warning': alert.type === 'warning', 'alert-success': alert.type === 'success', 'alert-info': !alert.type || alert.type === 'info', 'toast-pf': $ctrl.toast, 'mar-left-sm': $ctrl.toast}">
```

Comment:
Not sure if we have a consensus on style in the console, but I def prefer the previous style:

```html 
<!-- preferred -->
<div
   id="foo"
   class="bar"
   ng-complicated-thing="break,it,up,again"></div>

<!-- not preferred, paragraph form gets hard to parse quickly -->
<div id="foo" class="bar" ng-complicated-thing="break,it,up,again"></div>
```
That and extremely long line lengths make it difficult to use multiple columns in a text editor.  Its nice to open JS `controller` next to HTML `view` and sometimes a JS `service` and be able to see everything.

We don't have a style guide tho, so I'll defer again to @jwforres @spadgett!

Reply:
I do like the idea of breaking these long lines down, but for some reason, it breaks the spec test. The compiled element does not get the proper class applied when it's formatted that way.

Reply:
oh it breaks the test?  what output do you get?